docs/EntityPlayer.md

```diff
@@ -1383,11 +1397,81 @@ ___
 ### Is·Black·Heart () {: aria-label='Functions' }
 [ ](#){: .alldlc .tooltip .badge }
 #### boolean IsBlackHeart ( int Heart ) {: .copyable aria-label='Functions' }
+This can be used instead of GetBlackHearts to figure out which soul hearts are black hearts.
+
+???- example "Example"
+    Imagine we have the following setup of hearts, where S is a soul heart and B is a black heart:
+
+    ```
+    B S S B B S S B B
+    ```
+
+    Each soul heart is composed of two halves. Only the odd numbers seem to trigger this function. Even numbers always return false. The following assumes that the indexing starts at 1 rather than 0, but regardless, it's the odd numbers that work here.
+
+    ```
+    B(1,2) S(3,4) S(5,6) B(7,8) B(9,10) S(11,12) S(13,14) B(15,16) B(17,18)
+    ```
+
+    ```lua
+    Isaac.GetPlayer():IsBlackHeart(1) -- returns true
+    Isaac.GetPlayer():IsBlackHeart(2) -- returns false (not useful)
+    Isaac.GetPlayer():IsBlackHeart(3) -- returns false
+    -- 1,7,9,15,17 all return true
+    ```
+
+    Quick code example to parse soul hearts vs black hearts:
+
+    ```lua
+    local tbl = {}
+    local player = Isaac.GetPlayer()
+    for i = 1, player:GetSoulHearts(), 2 do -- 18
+      table.insert(tbl, player:IsBlackHeart(i) and 'B' or 'S')
+    end
+    if player:GetSoulHearts() % 2 ~= 0 then
+      tbl[#tbl] = string.lower(tbl[#tbl]) -- half heart at end
+    end
+    print(table.concat(tbl, ' ')) -- B S S B B S S B B
+    ```
 
 ___
 ### Is·Bone·Heart () {: aria-label='Functions' }
 [ ](#){: .alldlc .tooltip .badge }
 #### boolean IsBoneHeart ( int heart ) {: .copyable aria-label='Functions' }
+This can be used to figure out the ordering of bone hearts amongst soul/black hearts.
+
+???- example "Example"
+    Imagine we have the following setup of hearts:
+
+    ```
+    BONE SOUL BONE BLACK BONE
+    ```
+
+    The indexing here is for whole hearts, and the index starts at 0.
+
+    ```lua
+    Isaac.GetPlayer():IsBoneHeart(0) -- returns true
+    Isaac.GetPlayer():IsBoneHeart(1) -- returns false
+    -- 0,2,4 all return true
+    ```
+
+    Quick code example to parse soul hearts vs black hearts vs bone hearts:
+
+    ```lua
+    local tbl = {}
+    local player = Isaac.GetPlayer()
+    for i = 1, player:GetSoulHearts(), 2 do -- 4
```

Comment:
Again. The inline comments don't make sense because they lack context. Please provide more informations.